### PR TITLE
Translated `react: Components > <Profiler>`

### LINKED ISSUE #414: `<Profiler>`
Page: https://id.react.dev/reference/react/Profiler

Assignee: @rimuhamu

### PR DESCRIPTION
**Pull request by rimuhamu**
Closes #414 

Translate the `react: Components > <Profiler>` page.
Page URL: https://react.dev/reference/react/Profiler

### CODE REVIEW

File: src/content/reference/react/Profiler.md [hunk 1]
```diff
@@ -32,18 +32,18 @@ Wrap a component tree in a `<Profiler>` to measure its rendering performance.
 
 #### Props {/*props*/}
 
-* `id`: A string identifying the part of the UI you are measuring.
-* `onRender`: An [`onRender` callback](#onrender-callback) that React calls it every time components within the profiled tree update. It receives information about what was rendered and how much time it took.
+* `id`: String yang mengidentifikasi bagian UI yang sedang Anda ukur.
+* `onRender`: Sebuah [`onRender` *callback*](#onrender-callback) yang dipanggil oleh React setiap kali komponen dalam pohon yang diprofilkan diperbarui. Menerima informasi tentang apa yang di*render* dan berapa lama waktu yang dibutuhkan.
```
Comment (r17x) [inline src/content/reference/react/Profiler.md:36]:
```suggestion
* `onRender`: Sebuah [`onRender` *callback*](#onrender-callback) yang dipanggil oleh React setiap kali komponen dalam pohon yang diprofilkan diperbarui. Menerima informasi tentang apa yang di-*render* dan berapa lama waktu yang dibutuhkan.
```

[Referensi](https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5772140)

File: src/content/reference/react/Profiler.md [hunk 2]
```diff
@@ -32,18 +32,18 @@ Wrap a component tree in a `<Profiler>` to measure its rendering performance.
...
 
-#### Caveats {/*caveats*/}
+#### *Caveats* {/*caveats*/}
 
-* Profiling adds some additional overhead, so **it is disabled in the production build by default.** To opt into production profiling, you need to enable a [special production build with profiling enabled.](https://fb.me/react-profiling)
+* Pembuatan *profiler* akan menambah waktu komputasi, sehingga **dinonaktifkan dalam *build* produksi secara *default*.** Untuk menggunakan *profiler* dalam produksi, Anda harus mengaktifkan sebuah [*build* produksi spesial dengan opsi pembuatan *profiler*.](https://fb.me/react-profiling)
```
Comment (r17x) [inline src/content/reference/react/Profiler.md:40]:
```suggestion
* Pembuatan *profiler* akan menambah waktu komputasi, sehingga **dinonaktifkan dalam *build* produksi secara bawaan.** Untuk menggunakan *profiler* dalam produksi, Anda harus mengaktifkan sebuah [*build* produksi spesial dengan opsi pembuatan *profiler*.](https://fb.me/react-profiling)
```

**Review by r17x — CHANGES_REQUESTED** [review]

File: src/content/reference/react/Profiler.md [hunk 3]
```diff
@@ -32,18 +32,18 @@ Wrap a component tree in a `<Profiler>` to measure its rendering performance.
...
 
 ---
 
-### `onRender` callback {/*onrender-callback*/}
+### `onRender` *callback* {/*onrender-callback*/}
 
-React will call your `onRender` callback with information about what was rendered.
+React akan memanggil `onRender` callback Anda dengan informasi tentang apa yang sudah di*render*.
```
Comment (r17x) [inline src/content/reference/react/Profiler.md:46]:
```suggestion
React akan memanggil `onRender` *callback* Anda dengan informasi tentang apa yang sudah di-*render*.
```

Idem https://github.com/reactjs/id.react.dev/pull/515#discussion_r1183645323

File: src/content/reference/react/Profiler.md [hunk 4]
```diff
@@ -53,20 +53,20 @@ function onRender(id, phase, actualDuration, baseDuration, startTime, commitTime
...
-* `startTime`: A numeric timestamp for when React began rendering the current update.
-* `endTime`: A numeric timestamp for when React committed the current update. This value is shared between all profilers in a commit, enabling them to be grouped if desirable.
+* `id`: String `id` prop dari pohon `<Profiler>` yang baru saja di*commit*. Ini memungkinkan Anda mengidentifikasi bagian mana dari pohon yang di*commit* jika Anda menggunakan beberapa *profiler*.
+* `phase`: `"mount"`, `"update"` atau `"nested-update"`. Hal ini memungkinkan Anda mengetahui, apakah pohon baru saja dipasang untuk pertama kali, atau di*render* ulang karena ada perubahan pada *props*, status, atau *hooks*.
+* `actualDuration`: Jumlah milidetik yang dihabiskan untuk me*render* `<Profiler>` dan turunannya untuk pembaruan terkini. Ini mengindikasikan seberapa baik *sub*-pohon menggunakan memoisasi (contohnya [`memo`](/reference/react/memo) dan [`useMemo`](/reference/react/useMemo)). Idealnya, nilai ini akan berkurang secara signifikan setelah pemasangan awal, karena banyak keturunan yang hanya perlu me*render* ulang jika *props* tertentu mereka berubah.
```
Comment (r17x) [inline src/content/reference/react/Profiler.md:58]: idem https://github.com/reactjs/id.react.dev/pull/515#discussion_r1183645323

File: src/content/reference/react/Profiler.md [hunk 5]
```diff
@@ -53,20 +53,20 @@ function onRender(id, phase, actualDuration, baseDuration, startTime, commitTime
...
+* `baseDuration`: Jumlah milidetik yang memperkirakan berapa lama waktu yang dibutuhkan untuk me*render* ulang seluruh *sub*-pohon `<Profiler>` tanpa pengoptimalan apa pun. Dihitung dengan menjumlahkan durasi *render* terbaru dari setiap komponen dalam pohon. Nilai ini memperkirakan biaya kasus terburuk dari *rendering* (misalnya, pemasangan awal atau pohon tanpa memoisasi). Bandingkan `actualDuration` dengan nilai ini untuk melihat apakah memoisasi berfungsi.
+* `startTime`: Stempel waktu numerik untuk mengetahui kapan React mulai me*render* pembaruan terkini.
```
Comment (r17x) [inline src/content/reference/react/Profiler.md:60]: idem https://github.com/reactjs/id.react.dev/pull/515#discussion_r1183645323

**Review by r17x — CHANGES_REQUESTED** [review]

**Comment by mhaidarhanif** [comment]
@rimuhamu Please fix and resolve this to be approved by @r17x

**Comment by rimuhamu** [comment]
done, please kindly review @r17x

**Comment by mhaidarhanif** [comment]
@rimuhamu has resolved the issues, @r17x can check and approve if everything's okay

**Review by mazipan — APPROVED** [review]

**Comment by mazipan** [comment]
@r17x check the review

**Review by r17x — APPROVED** [review]

**Pull request merged**